OpenRA.Mods.RA/Traits/Mine.cs

```diff
@@ -52,6 +52,9 @@ void INotifyCrushed.OnCrush(Actor self, Actor crusher, HashSet<string> crushClas
 
 		bool ICrushable.CrushableBy(Actor self, Actor crusher, HashSet<string> crushClasses)
 		{
+			if (!crusher.Info.HasTraitInfo<MineImmuneInfo>() && self.Owner.Stances[crusher.Owner] == Stance.Ally && info.AvoidFriendly)
```

Comment:
If I understand correctly, INotifyCrushed.OnCrush happens when unit is already on the tile with a mine. ICrushable.CrushableBy is checked prior to unit entering a tile, preventing friendly units from moving over mines - that's why I added it. One of the lines should be removed, but I think it needs to be the one in OnCrush - to avoid situations where one unit chases another over a minefield, and only one of them blows up, just because mines are friendly.
Should I remove that one, to complete PR?